ops/ruby/devops-framework/DEVOPS_FRAMEWORK.md

```diff
@@ -0,0 +1,78 @@
+# DevOps Framework
+
+## Purpose
+This is a small but flexible framework for automating DevOps actions and ensuring they are
```

Comment:
Is the proposal to move entirely to this new framework, including for our development scripts?